utils/land-patterns.stanza

```diff
@@ -911,22 +966,22 @@ public pcb-landpattern INDC1005X55 :
 
 public pcb-landpattern CAPT1005X55 :
   make-ipc-two-pin-landpattern("0402")
-  layer(Silkscreen("pol", Top)) = PlusSymbol(-1.7, 0.0)
+  draw-polarity-marker(self.p[1])
   ref-label()
 
 public pcb-landpattern CAPT1608X80 :
   make-ipc-two-pin-landpattern("0603")
-  layer(Silkscreen("pol", Top)) = PlusSymbol(-2.2, 0.0)
+  draw-polarity-marker(self.p[1])
   ref-label()
 
 public pcb-landpattern CAPT2012X125 :
   make-ipc-two-pin-landpattern("0805")
-  layer(Silkscreen("pol", Top)) = PlusSymbol(-2.4, 0.0)
+  draw-polarity-marker(self.p[1])
   ref-label()
 
 public pcb-landpattern CAPT3216X160 :
   make-ipc-two-pin-landpattern("1206")
-  layer(Silkscreen("pol", Top)) = PlusSymbol(-3.0, 0.0)
+  draw-polarity-marker(self.p[1])
```

Comment:
See later comment.

Reply:
Thanks for point out that these are actually calling: make-ipc-two-pin-landpattern instead of the newer make-two-pin-chip-landpattern.